packages/orbit-components/src/Checkbox/Checkbox.ct.tsx

```diff
@@ -1,12 +1,22 @@
 import * as React from "react";
 import { test, expect } from "@playwright/experimental-ct-react";
 
-import CheckboxStory from "./Checkbox.ct-story";
+import { CheckboxStory, CheckboxWithTooltipStory } from "./Checkbox.ct-story";
 
 test.describe("visual Checkbox", () => {
   test("Checkbox", async ({ mount }) => {
     const component = await mount(<CheckboxStory />);
 
+    component.getByRole("button").hover();
+    await expect(component).toHaveScreenshot();
+  });
+
+  test("Checkbox - active tooltip on mobile", async ({ mount, browserName }) => {
```

Comment:
I added this test to distinguish click on tooltip for mobile devices (hover doesn't work for mobile as the tooltip is like "modal window")

Reply:
Is this behaviour for darwin/linux random?

You are mentioning that `tablet` and `largeMobile` work differently, however there are several more discrepancies for other viewports as well. E.g. 

- `readOnly` has missing also `smallMobile` for linux (next to `tablet` and `largeMobile`)
- `hasError` has missing snapshots for both linux and darwin for `smallMobile` and weirdly on darwin for `mediumMobile` (it's there for linux)
- `checked` has it missing also for `mediumMobile` for darwin and `smallMobile` for linux (next to `tablet` and `largeMobile`)
- `disabled` has all snapshots correct

This randomness is worrying :(